src/main/java/com/julienviet/pgclient/codec/DataType.java

```diff
@@ -266,10 +287,24 @@ private static long decodeInt(int len, ByteBuf buff) {
   }
 
   private static Object decodeJson(String value) {
-    if(value.charAt(0)== '{') {
+    if(value.indexOf('{') != -1) {
```

Comment:
you should keep using `charAt(0)`, `indexOf` will do a full scan of the string when it does not find the curly brace

Reply:
The reason I did this because JSON doesn't trim spaces but JSONB does.